src/platform/ESP32/CHIPDevicePlatformConfig.h

```diff
@@ -29,6 +29,8 @@
 
 #define CHIP_DEVICE_CONFIG_LWIP_WIFI_STATION_IF_NAME "st"
 
+#define CHIP_DEVICE_CONFIG_ESP32_BLE_ERROR_MIN 1100000
+#define CHIP_DEVICE_CONFIG_ESP32_BT_ERROR_MIN 1200000
```

Comment:
Here as well, either use a common macro (as only either of Bluedroid or NimBLE will be enabled at one time) or use NIMBLE / BLUEDROID in the macro name

Reply:
Done. Kept the common macro `CHIP_DEVICE_CONFIG_ESP32_BLE_ERROR_MIN` .